tools/flang2/utils/upper/upperilm.in

```diff
@@ -683,10 +745,21 @@ QUFUNCA/QFUNCA	num	num	ilm	args
 PQUFUNCA/PQFUNCA	num	num	sym	ilm	args
 QUVFUNCA/QVFUNCA	num	num	sym	ilm	sym	args
 QABS    ilm
+QFLOOR  ilm
```

Comment:
missing #ifdef?

Reply:
The `#ifdef` had already been added on line 706 by commit 46ec115b.